projects/plugins/jetpack/changelog/fix-wpcom-api-warning-autoupdate

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: other
+Comment: This fixes a warning, nothing special here
```

Comment:
I'd like to point out that those changelog messages end up on the [release notes](https://github.com/Automattic/jetpack-production/releases), so it should describe what is the purpose of the contribution.

You can check an example here: https://github.com/Automattic/jetpack/pull/26810

The linked PR's changelog text matches exactly the first entry on the "Other changes" list: https://github.com/Automattic/jetpack-production/releases.

I'd recommend redacting this changelog entry 👍